exposure-notification/src/main/resources/application.yml

```diff
@@ -75,6 +75,7 @@ covid19:
     data-cache:
       enabled: true
       status-duration: PT5M
+      file-duration: PT10S
   maintenance:
```

Comment:
I'm thinking should we cache longer? Batch with the same batchId shouldn't change and we'll need to be prepared that all available batches are in the memory at the same time anyway.

Reply:
We certainly can increase this, but I was thinking that it could cause more trouble than help. Mostly we get this situation as a single burst and then it disappears as the load balancers caches the file. After that, this cache becomes irrelevant.

It's true, the batch won't change, but if it hasn't been written yet, it may get written very soon and the cache will continue returning empty until it expires, meaning that the other readers each continue building the batch directly from DB... which would also work and shouldn't even happen as the batch should be there already. So either way works, I just figured that it's not needed after the short interval and this means that the situation is re-checked from disk without too much delay.

Reply:
Yep, let's keep in in that way and we can tune it later if needed.